core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/util/deprecationMessages.kt

```diff
@@ -112,6 +75,12 @@ private const val message_0_15 = "Will be removed in 0.15."
 
 // endregion
 
+// region WARNING in 0.15, ERROR in 0.16
+
+private const val message_0_16 = "Will be removed in 0.16."
```

Comment:
MESSAGE_0_16? for constants (no needs to fix, but for future)

Reply:
it's a private const so it doesn't really matter (the linter doesn't complain), but we could change it in the future if you want to